tests/unit/llms/providers/test_bam.py

```diff
@@ -36,3 +36,68 @@ def test_basic_interface(provider_config):
     llm = bam.load()
     assert isinstance(llm, LangChainInterface)
     assert bam.default_params
+
+
+def test_params_handling(provider_config):
+    """Test that not allowed parameters are removed before model init."""
+    config.init_empty_config()  # needed for checking the config.dev_config.llm_params
+
+    # first three parameters should be removed before model init
+    # rest need to stay
```

Comment:
comment says the first 3 should be removed, but the test logic expects min/max_new_tokens to be there.

And nothing checks for the temperature+verbose parameters

Reply:
fixed @bparees